### DISCUSSION TITLE
How to install fooocus on a linux (ubuntu) wihtout internet access? What dependencies should i move?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#3481] [Q&A] [answered]
How to install fooocus on a linux (ubuntu) wihtout internet access? What dependencies should i move?

**Comment by mashb1t** [comment, marked as answer]
Fooocus does not provide a bundled linux file with all dependencies included.
I'd suggest to install it on a machine with internet access by setting up an embedded python 3.10 and installing all dependencies there, then moving the files to the offline PC.
